+dj/Relvar.m

```diff
@@ -92,23 +92,22 @@ function cleanup(self)
                                 @(x) strcmp(x.from, rels(ix).fullTableName), ...
                                 self.schema.conn.foreignKeys, 'uni', true);
                             fks = self.schema.conn.foreignKeys(fk_index);
-                            if ~fks.aliased
+                            if ~all(fks.aliased)
```

Comment:
This line still doesn't work for us. Should read `if ~all([fks.aliased])`

Reply:
Hey @zfj1, could you elaborate on why this change was necessary on your end? We would like to recreate the issue.

Reply:
If a table has multiple foreign keys, then `fks` will be a `1xN` struct array, e.g.
```
fks = 

  1×3 struct array with fields:

    attrs
    ref
    ref_attrs
    from
    primary
    multi
    aliased
```
In this case, `fks.aliased` will return multiple outputs rather than an array of outputs, e.g.,
```
fks.aliased

ans =

  logical

   0


ans =

  logical

   0


ans =

  logical

   0
```
In this case, `all(fks.aliased)` will return an `Too many input arguments` error, since you are technically calling `all(false,false,false)`. By calling `all([fks.aliased])` MATLAB interprets the call as `all([false,false,false])` which will be handled correctly and return `false`.

Reply:
@zfj1 , The test case and implementation has been included in https://github.com/datajoint/datajoint-matlab/pull/395.

Turns out `all(fks.aliased)` works with only two arguments, but not three or more, which we weren't testing for. Thanks for catching this.